core/controllers/feedback.py

```diff
@@ -282,3 +282,29 @@ def post(self):
         email_manager.send_suggestion_email(
             exploration.title, exploration.id, suggestion.author_id,
             exploration_rights.owner_ids)
+
+
+class InstantFeedbackEmailHandler(base.BaseHandler):
+    """Handles task of sending feedback message emails instantsly."""
+
+    def post(self):
+        payload = json.loads(self.request.body)
+        user_id = payload['user_id']
+        reference = payload['reference']
```

Comment:
Let's try to distinguish more clearly between a domain object and a dict representing an object -- cf. the convention to use exploration vs exploration_dict. In line with that, I suggest calling this `reference_dict`.